horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/user/KeycloakUserBackend.java

```diff
@@ -417,6 +417,9 @@ public List<UserService.UserData> machineAccounts(String team) {
             String prefix = getTeamPrefix(team);
             return keycloak.realm(realm).roles().get(prefix + Roles.MACHINE).getUserMembers(0, Integer.MAX_VALUE).stream()
                     .map(KeycloakUserBackend::toUserInfo).toList();
+        } catch (NotFoundException ex) {
+            LOG.warnv("Unable to list machine accounts for team {0}", team);
```

Comment:
If this is an expected scenario, i.e. if a team does not have machine accounts, then I think the asppropriate log level should be `debug`

Reply:
>Therefore I'd propose to catch the NotFoundException and either log it at DEBUG level or keep it as WARNING but without the stacktrace.

yeah as I mentioned in the description, that was exactly my doubt.. whether debug or warning.
I agree that given it is an expected scenario, DEBUG is much more suitable here

Reply:
Updated